### PR TITLE
simple code update

### PR DESCRIPTION
**Pull request by chenweisomebody126**
# Update the comments and python logic 

Fixes # (issue)

## Who can review?

Community members can review the PR once tests pass. Tag maintainers/contributors who might be interested:

### CODE REVIEW

**Review by hwchase17 — COMMENTED** [review]
example notebook would be really helpful!

File: langchain/graphs/__init__.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 from langchain.graphs.neo4j_graph import Neo4jGraph
 from langchain.graphs.networkx_graph import NetworkxEntityGraph
 
-__all__ = ["NetworkxEntityGraph", "Neo4jGraph"]
+__all__ = ["NetworkxEntityGraph", "Neo4jGraph", "NebulaGraph"]
```
Comment (leo-gan) [inline langchain/graphs/__init__.py:5]: Shouldn't we also add an import for the `NebulaGraph`?
like this:
`from langchain.graphs.nebulagraph import NebulaGraph`

File: langchain/graphs/nebula_graph.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        password: str = "nebula",
+        address: str = "127.0.0.1",
+        port: int = 9669,
+        session_pool_size: int = 30,
+    ) -> None:
+        """Create a new NebulaGraph wrapper instance."""
+        try:
+            import nebula3
+        except ImportError:
+            raise ValueError(
```
Comment (leo-gan) [inline langchain/graphs/nebula_graph.py:35]: should be
`raise ImportError`

**Comment by chenweisomebody126** [comment]
- [ ]

**Pull request closed without merging**